README.md

```diff
@@ -1 +1,61 @@
-# groot
+# Project Groot 🌱
+
+Project Groot is an open-source garden management system. In its core, Project Groot is a regular automatic plant growing system: it reads data from humidity and light sensors and decides when to give the plants more water, less water, more light or less light. What differentiates Project Groot from other garden management systems, is that it allows users to write their own growth plans for different plants. Users can then share their growth plans through the Groot network and users can also download and review plans from other users. Project Groot can also manage various plants with different sensors and growth plans at the same time.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine.
+
+### Prerequisites
+
+What things you need to install the software and how to install them:
+
+
+  * Download and Install [Vagrant](https://www.vagrantup.com/) and [Virtual Box](https://www.virtualbox.org/) on your local computer
+  * Clone/ Download the zip file from the Groot repository to your local computer
+
+### How to Spin/Tear Down The Virtual Environment In General
```

Comment:
No need to specify how vagrant works. You can always add a link to vagrant documentation.

Reply:
Take out the whole ### How to Spin/Tear Down The Virtual Environment In General section?